test/object_store/test_backup.py

```diff
@@ -11,14 +11,16 @@
 from test.object_store.conftest import get_s3_resource
 from test.topology.conftest import skip_mode
 from test.pylib.util import unique_name
```

Comment:
Please test all scopes, not just restore.

We need sanity checks (for example, that RF=#racks for rack restore), and tests for those checks.

Reply:
Done.
Single test run is ~1 minute, so I didn't test _all_ possible combinations within given limits (not to revitalize #17764 frustration) and instead parametrized test with some static topologies that cover all scopes